### DISCUSSION TITLE
juice-shop doesn't start

### DISCUSSION THREAD

**Discussion by kkyaa** [community/community#77400] [New to GitHub]
Hello, I would like to know why juice shop doesn't start. I screenshot the error. If someone could help me, thank you.

![Screenshot_2023-11-30_03_50_47](https://github.com/community/community/assets/151428199/20408cf3-fe63-44cc-bf59-f52d633f12ea)

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
